### ISSUE TITLE
Invalid value for background does not throw error

### ISSUE THREAD

**Issue by Pranomvignesh** [CSSLint/csslint#769]
I am using csslint npm package -**v1.0.5**
This is my test code .

```javascript
var csslint = require("csslint");
var linted = csslint.CSSLint.verify(`
.sample{
    background:efwf;
}
`);
console.log(linted.messages)
```
Output : 
```shell
[]
```
I have tried this in [csslint website](http://csslint.net/)
<img width="1065" alt="Screen Shot 2020-07-13 at 6 28 24 PM" src="https://user-images.githubusercontent.com/29809906/87307811-db386200-c537-11ea-8699-b736a35ce075.png">

When i tried the same in https://jigsaw.w3.org/css-validator/validator, I am getting expected results. 

<img width="1631" alt="Screen Shot 2020-07-13 at 6 28 54 PM" src="https://user-images.githubusercontent.com/29809906/87307823-dffd1600-c537-11ea-8bc8-5de68531ca48.png">

Do i have to **configure something in csslint** to achieve the same as i expect ? Correct me if i am wrong.
